images/linux/scripts/installers/1804/powershellcore.sh

```diff
@@ -8,6 +8,8 @@
 source $HELPER_SCRIPTS/document.sh
 
 LSB_RELEASE=$(lsb_release -rs)
```

Comment:
Where do we use this variable?

Reply:
variable has been removed.
commit: andy-mishechkin@aca1c5a